src/AddTodoForm.tsx

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
-
-const AddTodoForm = () => {
+interface AddTodoFormProps {
+    onAddTodo: (title: string) => void;
+}
+const AddTodoForm = ({onAddTodo}: AddTodoFormProps) => {
+    const handleAddTodo = (e: React.SyntheticEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        const inputField = e.currentTarget.elements.namedItem("title") as HTMLInputElement;
```

Comment:
nit: could use a hook here rather than having to fish out the `title` like this. Though this also works